### DISCUSSION TITLE
Kafka source with a Kerberos Keytab authentication

### DISCUSSION THREAD

**Discussion by Shogobg** [vectordotdev/vector#6096] [Q&A] [answered]
Hello everyone,

I have a Kafka server with Kerberos GSSAPI authentication using a keytab file.
I was looking about how to set up vector to use the Keytab file to authenticate with Kafka, but i couldn't find anything in the documentation.

Can anyone help me with this?

**Comment by jszwedko** [comment, marked as answer]
I'm not super familiar with kerberos, but you can pass options directly to `librdkafka` via things like:

```
[sources.kafka]
type = "kafka"
librdkafka.some_option = "foo"
```

I do see some mention of kerberos and keytab in the available options: https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md

**Reply by Shogobg** [reply]
Thank you! I think that will work.

**Reply by huanghuangzym** [reply]
> 谢谢！我觉得这样可行。

how this problem resolved?
